src/store/concepts/lists/endpoints.js

```diff
@@ -0,0 +1,4 @@
+export const getListsEndpoint = accountId => `/account/${accountId}/lists`
```

Comment:
endpoint is the combination of request method + request url. in your case this constants are just routes